application/values-test.yaml

```diff
@@ -1286,22 +1293,22 @@ grafanaDashboard:
         matchLabels:
           app: test-2
 
-cronJob: 
+cronJob:
```

Comment:
Add `priorityClass` for Cronjob for the test

Reply:
@kseniyashaydurova
Done